src/main/java/no/unit/nva/handlers/GetUserHandler.java

```diff
@@ -0,0 +1,32 @@
+package no.unit.nva.handlers;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import no.unit.nva.database.UserDb;
+import nva.commons.exceptions.ApiGatewayException;
+import nva.commons.handlers.ApiGatewayHandler;
+import nva.commons.handlers.RequestInfo;
+import nva.commons.utils.Environment;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GetUserHandler extends ApiGatewayHandler<Void, UserDb> {
```

Comment:
Do you intend to have more than one Lambda in this project?

Reply:
Yes. GetUser, AddUser, UpdateUser, GetRole, and maybe AddRole and UpdateRole